int16 arm_zm
int16 arm_bm
int16 arm_am
int16 arm_wr
int16 arm_wp
bool qdcl
